### ISSUE TITLE
SSR support?

### ISSUE THREAD

**Issue by ralphilius** [torusresearch/OpenLoginSdk#65]
I'm trying the `openlogin` with NextJS, but this error keep showing even I tried to exclude it on client side.

```
ReferenceError: window is not defined
```

**Comment by DedaDev** [comment]
Same problem here.

**Comment by chaitanyapotti** [comment]
This is fixed in 1.5.0

**Comment by jdnichollsc** [comment]
@chaitanyapotti I'm testing this SDK with [RemixJS](https://github.com/remix-run/remix) but I'm getting the below error:

```
index.js:6 Uncaught ReferenceError: process is not defined
    at ../../node_modules/@toruslabs/openlogin-jrpc/node_modules/pump/index.js (index.js:6:29)
    at __require2 (chunk-ZMEXVS3E.js:47:50)
    at randomId.ts:3:39
```

Do you have examples with Server Side Rendering using these React Frameworks? :)

Thanks for your collaboration! <3
